subprojects/jvm-services/src/test/groovy/org/gradle/internal/jvm/inspection/DefaultJvmMetadataDetectorTest.groovy

```diff
@@ -351,6 +375,7 @@ class DefaultJvmMetadataDetectorTest extends Specification {
          'os.arch': "x86_64",
          'java.vm.name': "OpenJDK 64-Bit Server VM",
          'java.vm.version': "${version}+36",
+         'java.vm.vendor': "BellSoft.",
```

Comment:
How did you find out those properties? Did you inspect the JVMs? Especially for this vendor name I find it surprising that there is a `.` at the end.